themes/survey/fruity_twentythree/views/subviews/survey/question_container.twig

```diff
@@ -18,7 +18,10 @@
     So we separate it
 #}
 <!-- Question {{ aQuestion.qid  }}  -->
-<div {{ aQuestion.attributes }} class="row {{ aQuestion.class }} {{ aQuestion.input_error_class }} {{ aSurveyInfo.class.questioncontainer }}" {{ aSurveyInfo.attr.questioncontainer }}>
+<div role="group" aria-labelledby="question-text-container-{{ aQuestion.SGQ }}" {{ aQuestion.attributes }} class="row {{ aQuestion.class }} {{ aQuestion.input_error_class }} {{ aSurveyInfo.class.questioncontainer }}" {{ aSurveyInfo.attr.questioncontainer }}>
     {{ include('./subviews/survey/question.twig') }}
 </div>
 <!-- End of question  {{ aQuestion.qid  }}  -->
+<script>
+document.querySelector('[aria-labelledby="ls-question-text-{{ aQuestion.SGQ }}"] > table')?.removeAttribute('role');
```

Comment:
The merge request https://github.com/LimeSurvey/LimeSurvey/pull/2888

was tested and approved by 2 A11Y specialized companies and tested in real situation.